test/qxOrderAction.js

```diff
@@ -0,0 +1,98 @@
+const {QubicTransferQXOrderPayload} = require("../dist/qubic-types/transacion-payloads/QubicTransferQXOrderPayload")
+const {PublicKey} = require("../dist/qubic-types/PublicKey")
+const {Long} = require("../dist/qubic-types/Long")
+const {QubicTransaction} = require("../dist/qubic-types/QubicTransaction")
+const {QubicDefinitions} = require("../dist/QubicDefinitions")
+
+const senderIdentity = ""
+const senderSeed = ""
+
+// CFB asset
+const assetIssuer = "CFBMEMZOIDEXQAUXYYSZIURADQLAPWPMNJXQSNVQZAHYVOPYUKKJBJUCTVJL"
+const assetName= 4343363
```

Comment:
this needs to be converted from string to uint64 by using a similar approach to what we have in the go library: https://github.com/qubic/go-qubic/blob/867ea2c1433d5d78266de49c77d7bb0ceba44500/sdk/qx/client.go#L81